### ISSUE TITLE
Merge the accessibility violations metric with violations metric

### ISSUE THREAD

**Issue by fniessink** [ICTU/quality-time#562]
Tasks:
- [x] Add the accessibility violation metric sources to the violations metric.
- [x] Automatically migrate the accessibility violation metric to the violation metric.
- [x] Give the old accessibility metrics a custom name "Accessibility violations" and a unit "accessibility violations" if the metric name and unit haven't been overridden before.
- [x] Update data model, including docs.
